zingolib/src/wallet/transactions.rs

```diff
@@ -38,3 +50,255 @@ impl TxMapAndMaybeTrees {
         self.witness_trees.as_mut().map(WitnessTrees::clear);
     }
 }
+
+pub struct ZingoAccount(AccountId, UnifiedFullViewingKey);
```

Comment:
I vote we move this thing somewhere else..   like a `zingo_account` mod.   It's only (minimally) implemented to fulfill an API constraint, and leaving it sitting around here makes the "transactions.rs" mod less readable, and more cluttered.